src/chunkserver/concurrent_apply/concurrent_apply.cpp

```diff
@@ -157,15 +157,6 @@ void ConcurrentApplyModule::Flush() {
     event.Wait();
 }
 
-ThreadPoolType ConcurrentApplyModule::Schedule(CHUNK_OP_TYPE optype) {
```

Comment:
consider change `ThreadPoolType` to `ApplyTaskType`, it's more meaningful

Reply:
done